app/Platform/Data/PlayerResettableGameAchievementData.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Data;
+
+use Spatie\LaravelData\Data;
+use Spatie\TypeScriptTransformer\Attributes\TypeScript;
+
+#[TypeScript('PlayerResettableGameAchievement')]
```

Comment:
These TypeScript annotations aren't relevant for the time being. They auto-generate types for the Inertia UI, but that is to come in a follow-up.